cli/app.go

```diff
@@ -173,6 +174,32 @@ var app = &cli.App{
 					ArgsUsage: "[organization]",
 					Action:    ListLocationsAction,
 				},
+				{
+					Name:  "api-key",
+					Usage: "work with an api-key for your location",
+					Subcommands: []*cli.Command{
+						{
+							Name:  "create",
+							Usage: "create an api key for your location",
+							Flags: []cli.Flag{
+								&cli.StringFlag{
+									Name:     dataFlagLocationID,
+									Required: true,
+									Usage:    "The location to create an api-key for",
+								},
+								&cli.StringFlag{
+									Name:  apiKeyCreateFlagName,
+									Usage: "the name of the location to create an api-key for. This will default to location-id-created_at_timestamp",
```

Comment:
Can you put the description of default behavior in `DefaultText` instead of `Usage`? Here and for all the new fields below where you describe default behavior.

Reply:
isn't default text what it automatically sets the value to? We can't do that statically ?